data_query.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX animal_idx ON visits (animal_id);
```

Comment:
@errea good call with the indexes.  I am quite sure that it helped but...

We are really missing the screenshots `after` you fixed the performance issues as requested [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=.%20Once%20you%20succeed%2C%20take%20a%20screenshot%20of%20the%20EXPLAIN%20ANALYZE%20result%20showing%20that%20time%20actually%20decreased.).

Please add them to this PR description.

Reply:
Ok, seen, would fix that